marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -543,6 +542,60 @@ public void upgradeProject() throws IOException {
         removeEmptyRangeElementIndexArrayFromFinalDatabaseFile();
     }
 
+    @Override
+    public void exportProject() throws IOException {
```

Comment:
So instead of throwing an IOException, let's catch any exception and rethrow it as a RuntimeException, e.g.

    catch(Exception ex) {
      throw new RuntimeException("Unable to export project, cause: " + ex.getMessage(), ex);
    }

If any error occurs, there's nothing the caller - in this case, a Gradle task - will be able to do other than pass the error onto the user. So we might as well do that in here.